cob_hardware_config/cob4-5/config/image_flip_params_head_cam.yaml

```diff
@@ -0,0 +1,23 @@
+# rotation mode (0=fixed angle as defined  by rotation_angle, 1=automatic upright rotation against gravity using reference coordinate frame with upwards directed z-axis, 2=as 1 but image rotation only in 90deg steps -> faster)
+# int
+rotation_mode: 0
```

Comment:
I'd like to test rotation_mode=1 as soon as cob4-1 is back next week. (rotation_mode=1 should work now with modifications to image_flip by @ipa-rmb https://github.com/ipa320/cob_perception_common/pull/65).